libs/community/langchain_community/embeddings/anyscale.py

```diff
@@ -0,0 +1,75 @@
+"""Anyscale embeddings wrapper."""
+from __future__ import annotations
+
+from typing import Dict
+
+from langchain_core.pydantic_v1 import Field, SecretStr, root_validator
+from langchain_core.utils import convert_to_secret_str, get_from_dict_or_env
+
+from langchain_community.embeddings.openai import OpenAIEmbeddings
```

Comment:
this class is deprecated. it may be better to just re-implement the functionality you need

Reply:
I tried to change it to `from langchain_openai import OpenAIEmbeddings`, but it raised tons of dependence issues. It seems that `langchain_openai` is not installed by default, so the community libs can NOT depends on it. Please guide how to solve it.